onchain/.changeset/stupid-doors-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cartesi/rollups": minor
+---
+
+add quorum factory and factory tests
```

Comment:
I think we should seek consistency in the changelog.
So, let me propose some major guidelines:

- Capitalize the first word of sentences
- Format code-like text (like `this`)
- Conjugate action verbs in the past tense

Also, tests won't make it to the `@cartesi/rollups` package, so they can be abstracted away from the changelog. We might, however, mention tests when they affect the actual contracts, for example, when they expose a new bug.

```suggestion
Added `QuorumFactory` contract and `IQuorumFactory` interface
```